ale/isd_generate.py

```diff
@@ -51,6 +51,19 @@ def main():
              "and the default strategy of replacing their final suffix with "
              ".json will be used to generate the output file paths."
     )
+    parser.add_argument(
+        "-r", "--radius",
+        type=float,
+        default=None,
+        help="Optional spherical radius (km) override.  If not specified, the default "
+             "radius values, as defined in the NAIF kernels, will be used.  "
+             "There often cases were the defined NAIF radii are set as a triaxial "
+             "when the IAU WGCCRE report recommends a best-fit sphere for a "
```

Comment:
Remove the comments about the default NAIF kernels as it is not correct. Instead mention that this is for when the radii you want are not programmatically available (could be NAIF kernels, could be the cube labels, could be some other data source).